src/service.ts

```diff
@@ -103,6 +105,7 @@ export class Service {
   constructor(config: ServiceConfig, options: ServiceOptions = {}) {
     this.baseUrl = config.baseUrl;
     this.apiEndpoint = config.apiEndpoint;
+    this.timeout = options.timeout!;
```

Comment:
I'd be nervous using the `!` here because it looks like it can actually be undefined

Reply:
If `options.timeout` is undefined the [default](https://github.com/googleapis/nodejs-common/blob/master/src/util.ts#L36) value for timeout will be used [later](https://github.com/googleapis/nodejs-common/blob/master/src/util.ts#L717) and not swapped with `undefined` [here](https://github.com/googleapis/nodejs-common/blob/master/src/util.ts#L744)